data/src/main/java/io/gnosis/data/models/Transaction.kt

```diff
@@ -19,23 +29,48 @@ sealed class Transaction {
 
     data class SettingsChange(
         override val status: TransactionStatus,
+        override val confirmations: Int?,
         val dataDecoded: DataDecodedDto,
         val date: String?,
         val nonce: BigInteger
-    ) : Transaction()
+    ) : Transaction() {
+        fun isChangeMasterCopy(): Boolean {
+            return "changeMasterCopy" == dataDecoded.method
+        }
+
+        fun isSetFallBackHandler(): Boolean {
+            return "setFallbackHandler" == dataDecoded.method
+        }
+
+        fun isModuleChange(): Boolean {
+            return "enableModule" == dataDecoded.method || "disableModule" == dataDecoded.method
+        }
```

Comment:
It can't be in the viewHolder, because these checks need to be done to create the viewHolder. I inlined these methods and created isComplete(tx) in TransactionViewModel.